### PR TITLE
feat: udi field type changed to object in equipment schema

### PR DESCRIPTION
**Pull request by SergeyTatarenko**

### CODE REVIEW

File: __schema__/equipment.graphql [hunk 1]
```diff
@@ -62,7 +62,7 @@ type Equipment implements Node {
   "Identifier of equipment in external system."
   externalId: String!
   "Array of unique identifiers assigned to the asset by the manufacturer."
-  udi: [String]
+  udi: [Udi]
```
Comment (kiurchv) [inline __schema__/equipment.graphql]: Usually we're using uppercase abbreviations
```suggestion
  udi: [UDI]
```

File: __schema__/equipment.graphql [hunk 2]
```diff
@@ -100,3 +100,15 @@ enum EquipmentStatus {
...
+Udi structure in Equipment
+"""
+type Udi {
+  "Udi identifier"
+  value: String!
+  "Udi type, is choosen from dictionary. The value should be present in the `EQUIPMENT_UDI_TYPE` dictionary."
+  type: String!
+  "The name of the assigner"
+  assignerName: String
+}
```
Comment (kiurchv) [inline __schema__/equipment.graphql]:
```suggestion
}

```

File: __schema__/equipment.graphql (hunk 1 above)
Comment (SergeyTatarenko) [reply __schema__/equipment.graphql]: It is needed to rename type Udi to UDI too?

Comment (kiurchv) [reply __schema__/equipment.graphql]: Sure

**Pull request merged**